aep/general/0205/aep.md.j2

```diff
@@ -1,5 +1,47 @@
 # Beta-blocking changes
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/50) for more
-information.
+APIs often release an Alpha version of their API in order to get early feedback
+from customers. This API is provisional and can change many times before the
+important feedback is incorporated and the API is made stable for Beta.
+
+Since the purpose of Alpha is to gather feedback, the API does not need to be
+perfect yet, and it's not strictly necessary for API authors to address every
+usability concern or address every point in the API standards. Often, API
+authors and API reviewers will not agree on the best design, and the best way
+to find out is by having users try out the API.
+
+However, once the feedback has been collected and the API is going to be
+promoted to Beta, usability concerns and style issues do need to be addressed.
+In order to ensure that these issues are not forgotten, they should be
+explicitly documented in the API.
+
+## Guidance
+
+If an API has usability concerns or violates API standards, and the present
+design should receive additional scrutiny before being carried through to the
+Beta version, there **must** be an internal comment linking to this document
+using its descriptive link ([aep.dev/beta-blocker]()) to ensure that the design
```

Comment:
```suggestion
using its descriptive link ([aep.dev/beta-blocking-changes]()) to ensure that the design
```

Reply:
or maybe `aep.dev/beta-blocker` could be a `redirect_from: /beta-blocker` in the yaml file?